XiEditor/AppDelegate.swift

```diff
@@ -257,8 +306,33 @@ class AppDelegate: NSObject, NSApplicationDelegate, XiClient {
         testWindow?.makeKeyAndOrderFront(self)
         testWindow?.contentView = TextPlaneDemo(frame: frame)
     }
+    
+    func updateRpcTracingConfig(_ enabled: Bool) {
+        guard let dispatcher = self.dispatcher else { return }
```

Comment:
I think it's better to crash here, if we ever don't have a dispatcher we'd like to find out asap.

Reply:
Discussed on IRC.  Needed here so that app doesn't crash on boot since the collectTracingSamplesEnabled will be initialized to true before the dispatcher is created (so that we capture the early samples that are recorded before we initialize the dispatcher).